citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/AsyncCitationManager.java

```diff
@@ -46,6 +48,11 @@ public AsyncCitationManager() {
         this.updateTaskTracker = new ConcurrentHashMap<>();
         this.deleteTaskTracker = new ConcurrentHashMap<>();
         this.deleteTaskTimestamps = new ConcurrentHashMap<>();
+        this.hiddenItemsSet = new HashSet<>();
```

Comment:
is that holding all hidden items from all groups/users? that doesn't seem to make a lot of sense. also, after a restart all of this is lost. hidden items need to be persisted with Zotero (probably using a new property).